src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs

```diff
@@ -226,10 +226,6 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                     {
                         member = typeof(Color).GetField("Empty");
                     }
-                    else if (c.IsSystemColor)
-                    {
-                        member = typeof(SystemColors).GetProperty(c.Name);
-                    }
```

Comment:
That we're able to delete this without changing any tests suggests we don't have any tests covering this functionality?